WIP BXMSDOC-8000 & BXMSDOC-8001: Update Kogito book as per 7.12 STP BAPLs (#72)
hmanwani-rh/kie-docs: doc-content/enterprise-only/getting-started-kogito/ref-kogito-microservices-app-examples.adoc

* `dmn-drools-quarkus-metrics` and `dmn-drools-springboot-metrics`: A decision service (on {QUARKUS} or Spring Boot) that enables and consumes the runtime metrics monitoring feature in {KOGITO}.
* `pmml-quarkus-example` and `pmml-springboot-example`: A decision service (on {QUARKUS} or Spring Boot) that uses PMML.

Process services::
* `process-scripts-quarkus` and `process-scripts-springboot`: A process service (on {QUARKUS} and Spring Boot) that invokes scripts within a process.
* `process-service-calls-quarkus` and `process-service-calls-springboot`: A process service (on {QUARKUS} and Spring Boot) that invokes service calls within a process.

> * `process-rest-service-call-quarkus` and `process-rest-service-call-springboot`:  A process service (on {QUARKUS} and Spring Boot) that invokes REST service calls within a process.

Reviewer: ```suggestion
* `process-rest-service-call-quarkus` and `process-rest-service-call-springboot`:  A process service (on {QUARKUS} and Spring Boot) that invokes REST service calls within a process using a service task.
```

Author: Done

Reviewer: This was added to the wrong examples, one line above. It should be at `process-rest-service-call-*` examples.

> * `process-rest-workitem-quarkus`: A process service on {QUARKUS} that invokes a remote REST call within a finding user process.

Reviewer: ```suggestion
* `process-rest-workitem-quarkus`: A process service on {QUARKUS} that invokes a REST call using a REST work item within a finding user process.
```

Author: Done

> * `process-rest-workitem-multi-quarkus`: A process service on {QUARKUS} that invokes two consecutive REST calls within a process.

Reviewer: ```suggestion
* `process-rest-workitem-multi-quarkus`: A process service on {QUARKUS} that invokes two consecutive REST work item calls within a process.
```

Author: Done
